### ISSUE TITLE
systemd timer unit executes at midnight instead of hourly

### ISSUE THREAD

**Issue by laolux** [systemd/systemd#17464]
**systemd version the issue has been seen with**
 > systemd 246 (v246.6-3.fc33)

**Used distribution**
 > Fedora 33

**Linux kernel version used** (`uname -a`)
 > Linux localhost 5.8.16-300.fc33.x86_64 #1 SMP Mon Oct 19 13:18:33 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**CPU architecture issue was seen on**
 > x86_64

**Expected behaviour you didn't see**
 > systemd timer unit gets scheduled every hour

**Unexpected behaviour you saw**
 > systemd timer unit gets scheduled next midnight, approx 6 hours from now

**Steps to reproduce the problem**
 > create a dummy service unit and a corresponding timer unit. Timer unit as follows:

    [Unit]
    Description=timer for testing unit

    [Timer]
    OnCalendar=hourly
    Unit=testingunit.service

    [Install]
    WantedBy=timers.target

Start with `systemctl start testingunit.timer` and the output is:

    NEXT                        LEFT          LAST                        PASSED       UNIT                         ACTIVATES                     
    Wed 2020-10-28 00:00:00 JST 5h 37min left Tue 2020-10-27 16:58:47 JST 1h 24min ago testingunit.timer                  testingunit.service

**Comment by anitazha** [comment]
This definitely sounds like a bug but I wasn't able to repro on 246.6 on Fedora 31 or 246.1 on CentOS 7 so I'm wondering if there's something missing here.

**Comment by laolux** [comment]
Problem solved, not a bug. Completely my mistake. I erroneously had put unit files in both, `/etc/systemd/system` as well as `/usr/lib/systemd/system` and precedence rules rules screwed me up. I had changed one timer unit and had forgotten about the other one. Turns out the forgotten one was used and had a setting of `daily` :-( This also explains why I could not reproduce this problem in a simplified container.
I apologize for the work I caused.
